.gitignore

```diff
@@ -103,5 +103,9 @@ dist
 # TernJS port file
 .tern-port
 
+# vscode
+.vscode
+*code-workspace
+
```

Comment:
I think we do not need these changes, It depends on people's environment setup and may vary.

Reply:
Hmmm... alright, but is that not what `.gitignore` is for? I believe you will find the similar lines in Fastify's `.gitignore`.

Reply:
You are right but in my opinion, environment-specific things should not be added to gitignore file. If it's project-specific things then good to go.

Reply:
I will revise the gitignore file accordingly. You need to ignore environment-specific files in your global gitignore.